### ISSUE TITLE
2.4.13 Focus Appearance: Figure 18 in Understanding is incorrect

### ISSUE THREAD

**Issue by mitchellevan** [w3c/wcag#3666]
### Current

https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance.html#non-text-contrast-fail-focus-appearance-pass

Current content:

![Screenshot: "Figure 18 This example passes Focus Appearance but fails Non-text Contrast; there is insufficient adjacent contrast between the focus indicator and the adjacent colors." A light blue button with a 3px thick border. The border is black in the unfocused state and light gray in the focused state.](https://github.com/w3c/wcag/assets/3124805/20bedd24-fb56-441c-8c20-c3d207ba205a)

Issue: The caption for figure 18 says it fails SC 1.4.11 Non-text Contrast. However, figure 18 actually does not fail SC 1.4.11 because, if the low-contrast colors (light blue and light gray) were changed to a single solid light color, the removal of the black border would still be enough to identify which of the three buttons is focused.

This is explained in [Understanding SC 1.4.11](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html) as follows. "If you remove the adjacent color which does not have sufficient contrast, can you still identify and understand [the information]?" And in the testing steps: "If the least-contrasting area is less than 3:1, assume that area is invisible, is the [information] still understandable?" Understanding SC 1.4.11 provides these explanations only in context of "Graphical Objects: Parts of graphical objects required to understand the content," but they apply equally to "User Interface Components: Visual information required to identify user interface components and states."

### Proposed

Provide a figure similar to one in @alastc's comment here: https://github.com/w3c/wcag/issues/1613#issuecomment-773410792
